config/prod/shinyserver-pro-prod-alb-v2.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision a load balancer for shiny.synapse.org
+template_path: remote/managed-alb-https-v2.yaml
+stack_name: shinyserver-pro-prod-alb-v2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastucture"
+  # The resource owner
+  OwnerEmail: "x.schildwachter@sagebase.org"
+
+  # load balancer configs
+  Ec2InstanceId: "i-0a41a3cbb9de74e9f"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
```

Comment:
maybe there's a cert you can reference using !stack_output_external as well?

Reply:
i guess that's a no?

Reply:
No, there's one for *.sagebase.org but not for *.synapse.org